node/service/src/lib.rs

```diff
@@ -1477,6 +1478,7 @@ fn revert_chain_selection(db: Arc<dyn Database>, hash: Hash) -> sp_blockchain::R
 	let config = chain_selection_subsystem::Config {
 		col_data: parachains_db::REAL_COLUMNS.col_chain_selection_data,
 		stagnant_check_interval: chain_selection_subsystem::StagnantCheckInterval::never(),
+		stagnant_check_mode: chain_selection_subsystem::StagnantCheckMode::PruneOnly,
```

Comment:
will pruning be ever called if check interval is `never`?

Reply:
No, it won't. But as far as I understand, this code is called merely for reversion of the chain selection and notihing else. Please correct me if I'm wrong.

Reply:
Oh, sorry, I missed some part of the diff.